docs/configuration-files.rst

```diff
@@ -0,0 +1,191 @@
+Anaconda configuration files
+============================
+
+The installer loads its default configuration from the Anaconda configuration files. The
+configuration can be modified by kernel arguments and cmdline options and the result is written
+into a temporary configuration file. The configuration is not supposed to change after that.
```

Comment:
I am not really sure that it will have no effect. If you really want to, you are able to change the runtime configuration (you can reload the temporary file from the code or change the temporary file and restart a DBus module), but the modules will be out of sync and it can have unexpected consequences, so I think that the current statement is correct.